source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
+additional information on upstream format versions.
 
-If the mix has many bundles, this step might take some time.
+Use the following steps to manually pull a mixer Docker container:
 
-Mixer automatically gathers the bundle definition files for the upstream
-bundles into a :file:`upstream-bundles` directory, and user bundles should
-be placed directly into :file:`local-bundles`. The local path is set in
-the `LOCAL_BUNDLE_DIR` variable in the :file:`builder.conf`. **Do not edit
-files in upstream-bundles.** Mixer automatically deletes the contents of
-the :file:`upstream-bundles` directory before repopulating the directory
-on-the-fly if a new version must be downloaded.
+#. Find the version of the container you need by viewing the tags for the
+   `clearlinux/mixer repo <https://hub.docker.com/r/clearlinux/mixer/tags/>`_
+   on Docker Hub.
 
+#. Pull the latest container version:
 
-Create an update
-================
+   .. code-block:: bash
 
-Create an update with the following command:
+      docker pull clearlinux/mixer:<upstream-format-version>
 
-.. code-block:: bash
+#. View local docker images:
 
-   mixer build update
+   .. code-block:: bash
 
-When the build completes, you can find the mix update content under
-:file:`/home/clr/mix/update/www/VER`. In our example, the update content is
-found in :file:`/home/clr/mix/update/www/{<MIXVERSION>}`. `<MIXVERSION>`
-is the defined mix version, which is 10 by default.
+      docker images
 
-.. code-block:: bash
+.. rst-class:: content-collapse
 
-   mixer build update 
+Configure Docker proxy info
+===========================
 
-Mixer creates all the content needed to make a fully usable mix with this
-step. However, only *zero packs* are automatically generated. Zero packs are
-the content needed to go from nothing to the mix version for which you just
-built the content.
+If needed, use these steps to configure the Docker proxy information.
 
-You can create optional *delta packs*, which allow the transition from one
-mix version to another, with the following command:
+Configure the Docker daemon proxies:
 
-.. code-block:: bash
+#. Create the Docker daemon proxy config file and add the
+   following entries, using your own proxy values:
 
-   mixer build delta-packs --to <MIX_VERSION> --from <PAST_VERSION>
+   .. code-block:: bash
 
-This command generates all delta packs for the bundles changed from
-`PAST_VERSION` to `MIX_VERSION`. Mixer cannot create delta packs for the
-first build because the update is from version 0. Version 0 implicitly has
-no content, thus mixer can generate no deltas.
+      sudo mkdir -p /etc/systemd/system/docker.service.d
+      sudo nano /etc/systemd/system/docker.service.d/http-proxy.conf
 
-For subsequent builds, you can run :file:`mixer-pack-maker.sh` to generate
-delta content between them, for example: 10 to 20.
+   .. code-block:: console
 
-Create an image
-===============
+      [Service]
+      Environment="HTTP_PROXY=[HTTP proxy URL]"
```

Comment:
Two comments:
1. Inconsistency in using [] vs <> for substitutions.
2. Please add "port number", i.e. Environment="HTTP_PROXY=[HTTP proxy URL]:[port number]"

Reply:
good catch - thanks!